File: EventServiceApi/EventServiceApi/Models/Transport.cs
using System.ComponentModel.DataAnnotations;
using EventServiceApi.ViewModels;

namespace EventServiceApi.Models
{
    public class Transport
    {
        [Key]
        public long Id { get; set; }

        [Required]
        [MaxLength(50)]
        public string StateNumber { get; set; }

        [Required]
        [MaxLength(50)]
        public string DriverFirstName { get; set; }

        [MaxLength(50)]
        public string DriverLastName { get; set; }

        [Required]
        [MaxLength(50)]
        public string DriverRequiredPhoneNumber { get; set; }

        [MaxLength(50)]
        public string DriverAdditionalPhoneNumber { get; set; }

        [Required]
        [MaxLength(50)]
        public string City { get; set; }

        [MaxLength(50)]
        public string District { get; set; }

        public TransportViewModel ConvertToViewModel()
        {
            TransportViewModel viewModel = new TransportViewModel()
            {
                TransportId = Id,
                TransportStateNumber = StateNumber,
                DriverFirstName = DriverFirstName,
                DriverLastName = DriverLastName,
                DriverRequiredPhoneNumber = DriverRequiredPhoneNumber,
                DriverAdditionalPhoneNumber = DriverAdditionalPhoneNumber,
                TransportCity = City,
                TransportDistrict = District
            };

            return viewModel;
        }

    }
}

File: app/src/main/java/com/example/tadbir/Transport.java
package com.example.tadbir;

import com.google.gson.annotations.SerializedName;

public class Transport {
    @SerializedName("Id")
    public long id;
    @SerializedName("StateNumber")
    public String stateNumber ;
    @SerializedName("DriverFirstName")
    public String driverFirstName ;
    @SerializedName("DriverLastName")
    public String driverLastName ;
    @SerializedName("AdditionalPhoneNumber")
    public String additionalPhoneNumber;
    @SerializedName("DriverRequiredPhoneNumber")
    public String driverRequiredPhoneNumber ;
    @SerializedName("DriverAdditionalPhoneNumber")
    public String driverAdditionalPhoneNumber ;
    @SerializedName("City")
    public String city ;
    @SerializedName("District")
    public String district;
}
